src/services/analytics/push.ts

```diff
@@ -34,7 +34,7 @@ export class Push {
         }
       }
 
-      await Promise.allSettled(
+      await Promise.all(
```

Comment:
`allSettled` is safer, as even if _one_ fails, then `all` will fail; if _one_ fails with `allSettled`, it will await the rest of the promises to finish